lib/appmetrics-prometheus.js

```diff
@@ -273,6 +262,267 @@ function stringHttpRequestDuration(httpURLDataList) {
   return content;
 }
 
+function stringProcessUptime() {
+  return [
+    '# HELP process_uptime_count_seconds The number of seconds for which the current Node.js process has been running',
+    '# TYPE process_uptime_count_seconds counter',
+    `process_uptime_count_seconds ${process.uptime()}\n`,
+  ];
+}
+
+function stringLoopData(latestLoopEvent) {
+  if (!latestLoopEvent) return;
+  return [
+    '# HELP event_loop_tick_min_milliseconds The shortest tick time in the event loop samples, in milliseconds',
+    '# TYPE event_loop_tick_min_milliseconds guage',
+    `event_loop_tick_min_milliseconds ${latestLoopEvent.minimum}`,
+
+    '# HELP event_loop_tick_max_milliseconds The longest tick time in the event loop samples, in milliseconds',
```

Comment:
https://prometheus.io/docs/concepts/metric_types/#summary

https://prometheus.io/docs/practices/histograms/



I got a bit lost in the docs... but that aside, I _think_ that when client side calculates this stuff, that it is intended to be exported as a `_summary`. I'd take a close read and see, maybe find an example?